### ISSUE TITLE
Unable to create account

### ISSUE THREAD

**Issue by leonprou** [btcsuite/btcwallet#493]
I'm trying to create account using: `setaccount address name` rpc.
 
I'm getting a response from btcwallet:

`-1: Request unsupported by btcwallet
`

Same goes for the `move` command.

**Versions:**
btcwallet version 0.7.0-alpha
btcd version 0.12.0-beta
btcctl version 0.12.0-beta

**Comment by stevenroose** [comment]
That means that the btcctl and API support is there only for backwards compatibility with bitcoind, while btcwallet doesn't really support that.

**Comment by leonprou** [comment]
@stevenroose Hey, thanks for answering
I want to have two different wallets of a same computer. I though accounts was the answer.

**Comment by stevenroose** [comment]
Yeah but I guess setting an address to another account doesn't work because of BIP32. 
I would guess that the command `createnewaccount "account"` and `getnewaddress ("account")` do work.

**Comment by jrick** [comment]
Correct.  Addresses and balances can not be freely moved between accounts.  Create an account first, and then generate addresses from it.

**Comment by leonprou** [comment]
@stevenroose, @jrick Thanks, `createnewaccount`  account does work 👍 

I guess`createnewaccount` is not standard command? I could not find it not in [bitcoin wiki](https://en.bitcoin.it/wiki/Original_Bitcoin_client/API_calls_list) and [developers refference](https://bitcoin.org/en/developer-reference#rpc-quick-reference).  There's a difference from bitcoin reference client, and it's better to be stated.

**Comment by jrick** [comment]
From the README:

> The JSON-RPC server exists to ease the migration of wallet applications from Core, but complete compatibility is not guaranteed. Some portions of the API (and especially accounts) have to work differently due to other design decisions (mostly due to BIP0044). However, if you find a compatibility issue and feel that it could be reasonably supported, please report an issue. This server is enabled by default.

Creating new accounts is an explicit operation as it requires deriving account keys from the coin type private key with an unlocked wallet.

**Comment by leonprou** [comment]
Thanks, my bad..
